aten/src/ATen/native/native_functions.yaml

```diff
@@ -1858,7 +1858,15 @@
 - func: cudnn_batch_norm(Tensor input, Tensor weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float exponential_average_factor, float epsilon) -> (Tensor, Tensor, Tensor, Tensor)
   dispatch:
     CUDA: cudnn_batch_norm
-  autogen: cudnn_batch_norm.out
+
+# To pass the check in `add_generated_native_functions`,
+# param `reserve` is set as a placeholder, just passing an empty tensor for it.
+# usage:
+# auto reserve = torch::empty({0}, torch::device(torch::kCUDA));
+# at::native::cudnn_batch_norm_out(..., epsilon, output, save_mean, save_var, reserve);
+- func: cudnn_batch_norm.out(Tensor input, Tensor weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float exponential_average_factor, float epsilon, *, Tensor(a!) out, Tensor(b!) save_mean, Tensor(c!) save_var, Tensor(d!) reserve) -> (Tensor(a!), Tensor(b!), Tensor(c!), Tensor(d!))
```

Comment:
I think the BC test failure is complaining that `out1`, `out2`, and `out3` are renamed to `save_mean`, `save_var`, and `reserve` here? Can you confirm if this is the case? @yewentao256 

Also cc @albanD. I think the new names are better. Is it OK to break BC here?

Reply:
@andrewor14  Yes, I've tested it locally, this issue was due to the parameter names.

Reply:
<img width="1084" alt="企业微信截图_4706edbc-cffc-49c1-bab1-3d65a4eb1de6" src="https://github.com/pytorch/pytorch/assets/44945378/df6f5392-182e-482f-adba-378c502da1ca">

This can pass the bc check.

Reply:
breaking bc for an op as used as this one would be a 6months process. So we most likely just want to stick with the bad name :D

Reply:
@albanD Done! BTW, please remove the label `Stale`